deployment/helm/templates/rbac.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: akri-agent-sa
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
we should update the version for this change

Reply:
Version bumped.